File: app-marketplace/marketplace-backend/Backend.Domain/DTO/FilterDTO.cs
using Backend.Domain.Helpers;

namespace Backend.Domain.DTO
{
    public class FilterDTO
    {
        public FilterDTO() 
        { 
            SearchFields = new List<SearchField>();
        }

        public List<SearchField>? SearchFields { get; set; }
        public Paging? Paging { get; set; }
    }
}

File: app-marketplace/marketplace-frontend-fix/src/app/models/DTO/FilterDto.ts
export interface Paging{
    page :number;
    itemsPerPage :number;
    orderBy :string | null;
    descending :boolean;
}

export interface SearchField{
    property :string;
    value :any;
    operator :string;
}

export interface FilterDto{
    paging :Paging;
    searchFields :SearchField[];
}